src/core/handle.ml

```diff
@@ -133,6 +134,68 @@ let handle_require_as : popt -> sig_state -> Path.t -> ident -> sig_state =
   let path_map = PathMap.add p id.elt ss.path_map in
   {ss with aliases; path_map}
 
+(** [data_proof] returns the datas needed for the proof script check
+   TODO this is going to changes with the homogenization between
+   definition and theorem *)
+let data_proof x a cmd impl expo pdata_expo pos ts pe prop mstrat d goals =
```

Comment:
Why do you have two expo arguments?

Reply:
expo is for the symbol that's going to be added to the signature

pdata_expo is the symbol exposition allowed to be used in tactics

Reply:
They are not the same?

Reply:
public definition (expo = Public) => public symbols allowed (pdata_expo = Public)
private definition (expo = Privat) => private symbols allowed (pdata_expo = Privat)
public theorem (expo = Public) => public and private symbols allowed (pdata_expo = Privat)

Reply:
I see. Then, it would be good to add this comment somewhere.

Reply:
Well, the function `data_proof` needs a comment too...

Reply:
fixed in e6a8788
some arguments have been packed in a type, others have been removed (they were redundant)